src/coffee/directives/api/models/parent/rectangle-parent-model.coffee

```diff
@@ -22,8 +22,8 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
         else if scope.bounds.getNorthEast? and scope.bounds.getSouthWest? #google format
             bounds = scope.bounds
         else
-          if bound?
-            $log.error "Invalid bounds for newValue: #{JSON.stringify scope.bounds}" #note if bouds is recursive this could crash
+          if scope.bounds?
```

Comment:
I am not sure, if that line is executed - the last `else` is never reached, also all other conditions here check `scope.bounds`.